Atomic/images.py

```diff
@@ -259,7 +259,8 @@ def _filter_include_image(self, image_obj):
                                  "Please choose from {}".format(var, filterables))
             if var == "type":
                 var = "str_backend"
-            if getattr(image_obj, var, None).lower() != value.lower():
+
+            if value.lower() not in str(getattr(image_obj, var, None)).lower():
```

Comment:
Yes, good point, I will fix that. Although I would like to note that for all the filterable items, if the user specifies a filter that does not exist, it would get caught and an error would be returned. So for all filters that get here, the image_obj would have already initialized that attribute on creation (which incidentally defaults to None). Thus hasattr would always be true.